libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteDetector.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.navigation.core.fasterroute
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.navigation.base.trip.model.RouteProgress
+
+object FasterRouteDetector {
+
+    fun isRouteFaster(newRoute: DirectionsRoute, routeProgress: RouteProgress): Boolean {
```

Comment:
Is the logic for faster route going to be included in a separate PR?

Reply:
Yeah this requirement was not captured in this ticket.

Reply:
Let's cut a follow up ticket then @abhishek1508 👍